### PR TITLE
Add crypto_box_seal and crypto_box_seal_open

### PR DESCRIPTION
**Pull request by briandfoy**
These are much more convenient than crypto_box_easy.

### CODE REVIEW

File: lib/Sodium/FFI.pm [hunk 1]
```diff
@@ -2,7 +2,7 @@ package Sodium::FFI;
 use strict;
 use warnings;
 
-our $VERSION = '0.009';
+our $VERSION = '0.010';
```
Comment (genio) [inline lib/Sodium/FFI.pm]: Can you put this one back, please? The tooling takes care of version bumps and the current Git version (0.009) is the next to be released.

File: lib/Sodium/FFI.pm [hunk 2]
```diff
@@ -490,6 +490,60 @@ our %function = (
...
+			}
+
+			my $msg = "\0" x ( length($cipher_text) - crypto_box_SEALBYTES );
+			my $ret = $xsub->( $msg, $cipher_text, length($cipher_text), $pk, $sk );
+            if ($ret != 0) {
+                croak("Some internal error happened");
+            }
+			return $msg;
+        }
+    ],
```
Comment (genio) [inline lib/Sodium/FFI.pm]: Can you please fix the indention here? Perltidy can help:

```bash
perltidy -pro=.perltidyrc lib/Sodium/FFI.pm -o foo.pm
```

You could copy this section from the newly created throwaway file, `foo.pm`. When you're done, you can trash foo.pm.

File: t/pub_key_crypto/box.t [hunk 3]
```diff
@@ -70,4 +72,24 @@ ok(crypto_box_BEFORENMBYTES, 'crypto_box_BEFORENMBYTES: got the constant');
     is($decrypted, $msg, 'crypto_box_open_easy: got the correct response');
 }
 
+subtest crypto_box_seal => sub {
+	my ($pub, $priv) = crypto_box_keypair();
```
Comment (genio) [inline t/pub_key_crypto/box.t]: indention

File: lib/Sodium/FFI.pm (hunk 1 above)
Comment (genio) [reply lib/Sodium/FFI.pm]: I should really add a CONTRIBUTING.md doc here so this is all makes sense as the dzil tooling can be easily overlooked. I'll try to get that done as soon as I can.

**Review by genio — CHANGES_REQUESTED** [review]
Thank you! I just have a few minor nitpicks and we'll need to document the two new functions

File: lib/Sodium/FFI.pm (hunk 1 above)
Comment (briandfoy) [reply lib/Sodium/FFI.pm]: Fixed

File: lib/Sodium/FFI.pm (hunk 2 above)
Comment (briandfoy) [reply lib/Sodium/FFI.pm]: Done.

File: t/pub_key_crypto/box.t (hunk 3 above)
Comment (briandfoy) [reply t/pub_key_crypto/box.t]: Done.

**Review by genio — APPROVED** [review]

**Pull request merged**